raiden-dapp/src/components/TokenInformation.vue

```diff
@@ -29,7 +29,7 @@
       <v-col cols="8">
         <div class="token-information__description">
           <span class="token-information__balance">
-            {{ token.balance | displayFormat(token.decimals) }}
+            {{ (token.balance || 0) | displayFormat(token.decimals) }}
```

Comment:
`decimals` can be undefined as well iirc? Not sure

Reply:
Yes they can but the filter already takes care of that.

Reply:
Ah ok 👍

Reply:
https://github.com/raiden-network/light-client/blob/a48b2259d0a934689b1bd30392cbc72395866f18/raiden-dapp/src/filters.ts#L34